react/src/index.tsx

```diff
@@ -284,13 +256,11 @@ const root = ReactDOM.createRoot(
   document.getElementById('react-root') as HTMLElement,
 );
 root.render(
-  <React.StrictMode>
```

Comment:
Why did you remove this code?

Reply:
my misake .. i modified it https://github.com/lablup/backend.ai-webui/pull/2615/commits/1704f898fc36b2629f9311518ff5d0f11c62db75#diff-8e8a93d52a7e6e8a4b91770c2de7cb6b500148e2855dc2f5bca6f97f8271bc06